test/TestProjects/ExactMatchFlattenInheritance/Generated/Extensions/ResourceGroupExtensions.cs

```diff
@@ -86,15 +85,12 @@ public static Pageable<AzureResourceFlattenModel2> GetAzureResourceFlattenModel2
         /// <param name="name"> The String to use. </param>
         /// <param name="parameters"> The AzureResourceFlattenModel2 to use. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentException"> <paramref name="name"/> is empty. </exception>
-        /// <exception cref="ArgumentNullException"> <paramref name="name"/> or <paramref name="parameters"/> is null. </exception>
+        /// <exception cref="System.ArgumentException"> <paramref name="name"/> is an empty string, and was expected to be non-empty. </exception>
```

Comment:
Curious: why remove `using System` and use the full namespace here?

Reply:
`using System` has been removed because there is no more code that requires it (previously it was `throw new ArgumentNullException` that required `using System`). I think Roslyn doesn't add `using` directive only for doc comments

Reply:
I see, it's strange it doesn't add the using directive for doc comments.  I am fine taking Roslyn's suggestion here, was just wondering if we could avoid code churn.